Tasks/AzurePowerShellV5/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,77 @@
+[CmdletBinding()]
```

Comment:
this logic just checks the path in case of folder or zip case. Post this again the existing logic of updating the PS module path will get executed, few steps here are redundant, can we ensure at least with folder and zip case we dont execute the logic twice ?

Reply:
Which steps you think are redundant?